src/pytest_springfield/README.md

```diff
@@ -0,0 +1,6 @@
+`dottedname` Module
+===================
+
+This module exists as a dummy module to help test imports of
+entity classes when using dotted-name strings for `EntityField`
+types.
```

Comment:
Seems OK, but the wrong place. `pytest_springfield` should only be used for a pytest plugin